foundation/foundation-upgrade/src/test/resources/specifications/v1_7_0/foundation-core-java-migration.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Foundation Core Java upgrade migration
+
+  Scenario: Upgrade Foundation Core Java upgrade migration
+    Given a Java file references the cdi package
+    When the 1.7.0 foundation core java package migration executes
+    Then the references are updated to com.boozallen.aissemble.cdi
+
+  Scenario: Upgrade Foundation Core Java upgrade migration
+    Given a Java file references the metadata package
```

Comment:
A: if all of these are the same except the package name is changing, it might make sense to convert these to a single Scenario Outline.

Reply:
Changed to a single scenario and file.